### ISSUE TITLE
The requirements.exe is so bad

### ISSUE THREAD

**Issue by glennchina** [sstzal/DiffTalk#25]
Can anyone share a useable requirements.txt? It has many conflicts and error.

**Comment by Vivek17290** [comment]
yes it is old and python 3.7.0 does not have the required versions of the libraries
